source/MaterialXRuntime/Private/PrvElement.cpp

```diff
@@ -14,16 +14,21 @@ namespace MaterialX
 
 const string PrvElement::PATH_SEPARATOR = "/";
 
-PrvElement::PrvElement(RtObjType objType, const RtToken& name) :
+PrvElement::PrvElement(RtObjType objType, PrvElement* parent, const RtToken& name) :
     PrvObject(objType),
+    _parent(parent),
     _name(name)
 {
 }
 
-void PrvElement::initialize()
```

Comment:
We'll need to talk about why this needs to be removed.

Reply:
Having this on the base class and naming it `initialize` to me implies this needs to be called on all elements after creation. Which I could not see a need for so I removed it. But let's talk more offline about this.